core/src/main/scala/org/apache/spark/storage/BlockManagerStorageEndpoint.scala

```diff
@@ -81,6 +81,8 @@ class BlockManagerStorageEndpoint(
     case ReplicateBlock(blockId, replicas, maxReplicas) =>
       context.reply(blockManager.replicateBlock(blockId, replicas.toSet, maxReplicas))
 
+    case MarkRDDBlockAsVisible(blockId) =>
```

Comment:
Can you add a comment for this as well ? Similar to `UpdateRDDBlockTaskInfo`, etc in `BlockManagerMasterEndpoint`

Reply:
Of course. Updated.